common/djangoapps/student/tasks.py

```diff
@@ -48,3 +55,17 @@ def send_activation_email(self, subject, message, from_address, dest_addr):
             exc_info=True
         )
         raise Exception
+
+
+@periodic_task(ignore_result=True, run_every=crontab(hour=settings.FEATURES.get('COURSE_REPORT_HOUR', '*')))
+def send_courses_report():
+    f = StringIO.StringIO()
+    csv_course_report(f)
+    email = mail.EmailMessage(
+        _('Course report'),
+        _('Course report in attachment...'),
+        settings.DEFAULT_FROM_EMAIL,
+        settings.FEATURES.get('USER_REPORT_EMAILS', [settings.CONTACT_EMAIL])
```

Comment:
Where is USER_REPORT_EMAILS set and what is are the values

Reply:
USER_REPORT_EMAILS can be set in server-vars.yml (transformed to lms.env.json on deploy). but can be omitted to use default platform e-mail address CONTACT_EMAIL (currently set to info@harambee.co.za)